cto_ai/daemon_request.py

```diff
@@ -43,6 +43,7 @@ def requester(body):
 get_all_state = _make_sync_requester("state/get-all")
 get_config = _make_sync_requester("config/get")
 get_all_config = _make_sync_requester("config/get-all")
+delete_config = _make_requester("config/delete")
```

Comment:
I think this needs to be `_make_sync_requester` if you want to return the boolean result. `_make_requester` doesn't read the JSON body.